src/handlers/records-write.ts

```diff
@@ -113,13 +126,49 @@ export class RecordsWriteHandler implements MethodHandler {
     return messageReply;
   };
 
+  /**
+   * Embeds the given data into the `encodedData` property to be stored in the MessageStore instead of DataStore
```

Comment:
Slight elaboration, since it took me a second to understand just reading the code.

```suggestion
   * Embeds the record's data into the `encodedData` property.
   * If dataStream is present, it uses the dataStream. Otherwise, uses the `encodedData` from the most recent RecordsWrite.
```